play/src/components/AnswerSelector.tsx

```diff
@@ -84,19 +85,23 @@ export default function AnswerSelector({
       </Typography>
       <Typography
         variant="body2"
-        sx={{ paddingLeft: '8px', paddingRight: '38px' }}
+        sx={{
+          paddingLeft: `${theme.sizing.extraSmallPadding}px`,
+          paddingRight: `${theme.sizing.largePadding}px`,
+        }}
       >
         {answerText}
       </Typography>
       <img
         src={imageMap[answerStatus]}
         style={{
           position: 'absolute',
-          right: isSubmitted ? '17px' : '16px',
-          width: '16px',
-          height: '16px',
-          paddingTop: '3px', // 16px + 3 px + 3px = 22px -> same line height as text, all per Figma
-          paddingBottom: '3px',
+          right: isSubmitted
+            ? `${theme.sizing.smallPadding + 1}px`
+            : `${theme.sizing.smallPadding}px`,
+          width: `${theme.sizing.smallPadding}px`,
+          height: `${theme.sizing.smallPadding}px`,
```

Comment:
You will probably want to avoid using sizing variables intended for centralized padding for the size of things like images. You can create other sizing variables, or consider whether this is a standardized size at all.

Reply:
makes a lot of sense, resolved to original values for now